docs/hub/unity-sentis.md

```diff
@@ -0,0 +1,66 @@
+# Using Sentis Models from Hugging Face
+
+[Unity Sentis](https://unity.com/products/sentis) is the inference engine which runs on Unity 2023 or above. 
+
+## Exploring Sentis Models in the Hub
+You will find `unity-sentis` models by filtering at the left of the [models page](https://huggingface.co/models?library=unity-sentis).
+
+All the Sentis models in the hub come with code and instructions to easily get you started using the model in Unity. All Sentis models have been validated to work so you can be sure they will run in Unity.
+
+To get more details about using Sentis you can read our [documentation](https://docs.unity3d.com/Packages/com.unity.sentis@latest). To get help from others using Sentis you can ask in our [discussion forum](https://discussions.unity.com/c/ai-beta/sentis)
+
+
+## Types of files
+Each repository will contain several types of files:
+
+* ``sentis`` files: These are the main model files which contain the neural networks which run on Unity.
+* ``ONNX`` files: This is an alternative format which you can include as well as, or instead of, the Sentis files.
```

Comment:
```suggestion
* ``ONNX`` files: This is an alternative format you can include as well as, or instead of, the Sentis files.
```

Reply:
```suggestion
* ``ONNX`` files: This is an alternative format you can include in addition to, or instead of, the Sentis files. It can be useful for visualization with third party tools such as [Netron](https://github.com/lutzroeder/netron).
```